config/locales/en.yml

```diff
@@ -126,7 +126,8 @@ en:
       invited_on: 'Invited on: %{date}'
       invitation_expired: 'Invitation expired'
       invite: 'Invite your loved ones!'
-      none: 'You currently have no allies! You can search for them or invite loved ones to join!'
+      none: 'You currently have no allies! You can search for them or invite
```

Comment:
So it looks like you're encountering failed tests because of how you created a new line.

Something like this:

```
Key: >
  This is a very long sentence
  that spans several lines in the YAML
  but which will be rendered as a string
  with only a single carriage return appended to the end.
```

Taken from this [StackOverflow post](https://stackoverflow.com/questions/3790454/how-do-i-break-a-string-over-multiple-lines).

Reply:
Fixed most of it, thanks :)
There seems to be an issue with line 436-437 when I split it into multiple lines. https://github.com/GayathriVenkatesh/ifme/blob/b2737c523d53a9fb14cca77c7b6bcd83009d1682/config/locales/en.yml#L436
Not sure why that's happening. Everything else works fine.

Reply:
Fixed most of it, thanks! 
There seems to be an issue with line 436 when I split it into multiple lines. Not sure why.
Everything else works fine.